gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/main/java/io/gravitee/apim/gateway/tests/sdk/AbstractWebsocketGatewayTest.java

```diff
@@ -89,7 +89,7 @@ protected void configureGateway(GatewayConfigurationBuilder gatewayConfiguration
     }
 
     @BeforeEach
-    public void setup(Vertx vertx) {
+    public void before(Vertx vertx) {
```

Comment:
to avoid problems with overriding maybe we can be more specific ?
```suggestion
    public void beforeEachWebsocketTest(Vertx vertx) {
```